src/s390x/mod.rs

```diff
@@ -17,5 +17,6 @@ pub mod zipl;
 
 pub use dasd::{dasd_try_get_sector_size, image_copy_s390x, prepare_dasd};
 pub use zipl::install_bootloader;
+pub use zipl::run_zipl;
```

Comment:
The distinction between these two functions isn't immediately obvious from the names.  Can we rename one or both?

Reply:
Added another commit that cleans things up a bit more.